frontend/src/global_styles/content/work_packages/tabs/_files.sass

```diff
@@ -29,6 +29,8 @@
 .op-files-tab,
 .op-attachments-list
   .op-files-tab--file-list
+    margin-bottom: 1rem
```

Comment:
Why is there a new `margin-bottom` of 16px on the list?

Reply:
Previously the `margin-bottom` was set on the `.work-package--attachments--files` (see [here](https://github.com/opf/openproject/pull/10925/files#diff-fc421eef86701064567f2f6d72548ffc4680b05bc081e2d34d7b668e9e27d974L138)). Since that element always existed, even if there were no attachments, there was always that additional spacing above the drop-zone. So I basically just moved the `margin-bottom` a few elements down the DOM.